docs/diagrams/classes/LdapConfiguration/LdapConfigurationBuilder.pu

```diff
@@ -0,0 +1,17 @@
+@startuml LdapConfigurationBuilder diagram
```

Comment:
c'e' un bisogno di usare un builder per LdapCOnfiguration?

Reply:
lo abbiamo implementato perchè in generale anche quelle informazioni possono essere incomplete o cambiare nel tempo